### DISCUSSION TITLE
How to sort/select large vs small grains in EBSD data

### DISCUSSION THREAD

**Discussion by JMB495** [mtex-toolbox/mtex#1432] [Ask Anything] [answered]
In MTex, Im trying to select the large/small grains independantly within an EBSD result, to see if there is a texture in the ReX crystals which differs from that in the large cold worked regions. I also wish to seperate out the plasticity of the two regions to see what influence this is having. I'm fairly confident i can do the second bit if I can get the first bit done.

All help greatly recieved.

Thanks

**Comment by kilir** [comment, marked as answer]
Hi
in a very generic way:
`ebsd_selected = ebsd(grains(grains.grainSize > xy))`
See here https://mtex-toolbox.github.io/SelectingGrains.html
Cheers,
Rüdiger

**Reply by JMB495** [reply]
thanks for this

**Comment by JMB495** [comment]
In the end I ended up with this;

% import a dataset by running mtex to import, or by opening a previously
% run version
% consider only indexed data for grain segmentation
ebsd = ebsd('indexed');
% perform grain segmentation
[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd);
% only the very big grains
big_grains = grains(grains.grainSize>100);
ebsd = ebsd(big_grains)

The way @kilir has suggested is alot sleeker :)
